src/babel/util.js

```diff
@@ -126,6 +162,10 @@ export function shouldIgnore(filename, ignore, only) {
   return false;
 }
 
+/**
+ * [Please add a description.]
```

Comment:
I think this was part of the rebase I just did, this function didn't exist previously